joomla/joomla-cms
# com_messages - Component naming 'Messaging'

The **Title** of this **Messaging component** is **different** in **Menu Names** and **Component Views**.

This PR **corrects the Menu Names and Component titles** so that the naming of the component is consequently the same.
### Testing Instructions

You'll need to create a couple of test users that have access to the back-end (e.g. Administrator or Super Administrator). Otherwise you cannot send Private Message and not see all screens.
#### Before the PR:

Backend: System > Global Configuration > under COMPONENT (left menu)
Configure Menu Title: **Messaging**

![configure-components-plural-single-before](https://cloud.githubusercontent.com/assets/1217850/18662622/6c199cb0-7f1b-11e6-874e-e002a672df04.png)

Backend: System > Global Configuration > under COMPONENT (left menu) click on Messaging
Configure Component Title: **Messages: Options**

![configure-components-plural-single-messaging-before](https://cloud.githubusercontent.com/assets/1217850/18662621/6c18166a-7f1b-11e6-9bb3-54454cbf424a.png)

Backend: Components
Menu Title: Components > **Messaging**
Menu Title: Components > Messaging > **New Private Message**

![configure-components-plural-single-messaging-menu](https://cloud.githubusercontent.com/assets/1217850/18662788/10a23e0e-7f1c-11e6-9163-9553ce3ea66c.png)

Backend: Components > Messaging > New
Write Message Title: **Private Messages: Write**

![configure-components-plural-single-messaging-component-write-before](https://cloud.githubusercontent.com/assets/1217850/18662619/6c14eeae-7f1b-11e6-8dbf-98d81e5d4318.png)

Backend: Components > Messaging
View Title: **Private Messages**

![configure-components-plural-single-messaging-component-read-before](https://cloud.githubusercontent.com/assets/1217850/18662620/6c15696a-7f1b-11e6-9a5e-5f6af4341161.png)

Backend: Components > Messaging > click on a new message
View Message Title: **Private Messages: Read**

![configure-components-plural-single-messaging-component-read-before2](https://cloud.githubusercontent.com/assets/1217850/18662947/c5f7f91a-7f1c-11e6-8807-a41487f43bf7.png)
